test/test_linalg.py

```diff
@@ -219,8 +219,8 @@ def select_if_not_empty(t, i):
             if a.numel() > 0:
                 for i in range(batch_size):
                     sol, residuals, rank, singular_values = ref(
-                        a_3d.select(0, i).numpy(),
-                        b_3d.select(0, i).numpy()
+                        a_3d.select(0, i).cpu().numpy(),
+                        b_3d.select(0, i).cpu().numpy()
```

Comment:
Is this change necessary?

Reply:
```
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```
Is what I ran into with this trace

```
  File "/home/ksm/pytorch/test/test_linalg.py", line 323, in test_linalg_lstsq
    check_correctness_scipy(a, b, res, driver, rcond)
  File "/home/ksm/pytorch/test/test_linalg.py", line 269, in check_correctness_scipy
    check_correctness_ref(a, b, res, scipy_ref, driver=driver)
  File "/home/ksm/pytorch/test/test_linalg.py", line 222, in check_correctness_ref
    a_3d.select(0, i).numpy(),
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```
After adding `gelss` as a driver for the test.

Reply:
But why was it passing before for the other inputs and the other drivers and not now?

Reply:
@lezcano Sorry for late reply, haven't been feeling too well. 

So this test is not failing for other drivers - it's only failing for SVD and GPU. The other driver on teh GPU, `gels` doesn't even run. 

Tangentially, I was told that I could comment out the `skipCUDAIfNoMagma` for this PR(I do not have magma built). Could you please confirm that is ok? I should probably also look into devcontainers for managing dependencies when contributing.

Reply:
Oh, now I see the skip down below. Oh well.